core/walkmod.xml

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE walkmod PUBLIC "-//WALKMOD//DTD" "http://www.walkmod.com/dtd/walkmod-1.1.dtd">
+<walkmod>
```

Comment:
Would it be possible to keep this file somewhere outside the top level? E.g. src/walkmod